EcoSoapBank/Impact/ImpactCoordinator.swift

```diff
@@ -10,9 +10,26 @@ import UIKit
 
 
 class ImpactCoordinator: FlowCoordinator {
-    var rootVC = ImpactViewController()
+    var rootVC = configure(UINavigationController()) {
+        $0.pushViewController(ImpactViewController(), animated: false)
```

Comment:
It doesn't matter so much here since we're `start`ing the coordinator right after it's initialized, but I think this `push` might fit better in the `start` method.

Reply:
Will do